### PR TITLE
`sage.graphs`: Doctest cosmetics

### PR DESCRIPTION
**Pull request by mkoeppe**
- Cherry-picked from #35095.

### :memo: Checklist

- [x] The title is concise and informative.
- [ ] The description explains in detail what this PR is about.
- [x] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation and checked the documentation preview.

### CODE REVIEW

File: src/sage/graphs/bipartite_graph.py [hunk 1]
```diff
@@ -2606,7 +2612,8 @@ class by some canonization function `c`. If `G` and `H` are graphs,
             ....:                      (1, 7), (1, 8), (2, 6), (2, 7), (2, 8),
             ....:                      (3, 4), (3, 7), (3, 8), (4, 9), (5, 9),
             ....:                      (6, 9), (7, 9)] )
-            sage: C, cert = B.canonical_label(partition=(B.left,B.right), certificate=True, algorithm='sage')
+            sage: C, cert = B.canonical_label(partition=(B.left,B.right),
```
Comment (gmou3) [inline src/sage/graphs/bipartite_graph.py]:
```suggestion
            sage: C, cert = B.canonical_label(partition=(B.left, B.right),
```

File: src/sage/graphs/bipartite_graph.py [hunk 2]
```diff
@@ -2620,7 +2627,8 @@ class by some canonization function `c`. If `G` and `H` are graphs,
 
             sage: G = Graph({0: [5, 6], 1: [4, 5], 2: [4, 6], 3: [4, 5, 6]})
             sage: B = BipartiteGraph(G)
-            sage: C = B.canonical_label(partition=(B.left,B.right), edge_labels=True, algorithm='sage')
+            sage: C = B.canonical_label(partition=(B.left,B.right),
```
Comment (gmou3) [inline src/sage/graphs/bipartite_graph.py]:
```suggestion
            sage: C = B.canonical_label(partition=(B.left, B.right),
```

File: src/sage/graphs/graph.py [hunk 3]
```diff
@@ -206,8 +206,7 @@
 
 - a list of edges::
 
-       sage: g = Graph([(1,3),(3,8),(5,2)])
-       sage: g
+       sage: g = Graph([(1,3),(3,8),(5,2)]); g
```
Comment (gmou3) [inline src/sage/graphs/graph.py]:
```suggestion
       sage: g = Graph([(1, 3), (3, 8), (5, 2)]); g
```

File: src/sage/graphs/graph.py [hunk 4]
```diff
@@ -790,14 +793,13 @@ class Graph(GenericGraph):
 
     #. List of edges, or labelled edges::
 
-          sage: g = Graph([(1,3),(3,8),(5,2)])
-          sage: g
+          sage: g = Graph([(1,3),(3,8),(5,2)]); g
```
Comment (gmou3) [inline src/sage/graphs/graph.py]:
```suggestion
          sage: g = Graph([(1, 3), (3, 8), (5, 2)]); g
```

File: src/sage/graphs/graph.py [hunk 5]
```diff
@@ -790,14 +793,13 @@ class Graph(GenericGraph):
...
           Graph on 5 vertices
 
-          sage: g = Graph([(1,2,"Peace"),(7,-9,"and"),(77,2, "Love")])
-          sage: g
+          sage: g = Graph([(1,2,"Peace"), (7,-9,"and"), (77,2, "Love")]); g
```
Comment (gmou3) [inline src/sage/graphs/graph.py]:
```suggestion
          sage: g = Graph([(1, 2, "Peace"), (7, -9, "and"), (77, 2, "Love")]); g
```

File: src/sage/graphs/graph.py [hunk 6]
```diff
@@ -836,8 +838,9 @@ class Graph(GenericGraph):
 
        If the igraph Graph has edge attributes, they are used as edge labels::
 
-           sage: g = igraph.Graph([(0,1),(0,2)], edge_attrs={'name':['a','b'], 'weight':[1,3]})  # optional - python_igraph
-           sage: Graph(g).edges(sort=True)                                                       # optional - python_igraph
+           sage: g = igraph.Graph([(0,1),(0,2)],                                # optional - python_igraph
```
Comment (gmou3) [inline src/sage/graphs/graph.py]:
```suggestion
           sage: g = igraph.Graph([(0, 1), (0, 2)],                             # optional - python_igraph
```

File: src/sage/graphs/graph.py [hunk 7]
```diff
@@ -836,8 +838,9 @@ class Graph(GenericGraph):
...
+           ....:                  edge_attrs={'name':['a','b'], 'weight':[1,3]})
```
Comment (gmou3) [inline src/sage/graphs/graph.py]:
```suggestion
           ....:                  edge_attrs={'name': ['a', 'b'], 'weight': [1, 3]})
```

File: src/sage/graphs/graph.py [hunk 8]
```diff
@@ -862,7 +865,8 @@ class Graph(GenericGraph):
           sage: {G:1}[G]
           Traceback (most recent call last):
           ...
-          TypeError: This graph is mutable, and thus not hashable. Create an immutable copy by `g.copy(immutable=True)`
+          TypeError: This graph is mutable, and thus not hashable.
+          Create an immutable copy by `g.copy(immutable=True)`
 
     When providing the optional arguments ``data_structure="static_sparse"`` or
     ``immutable=True`` (both mean the same), then an immutable graph results. ::
```
Comment (gmou3) [inline src/sage/graphs/graph.py]:
```suggestion
    ``immutable=True`` (both mean the same), then an immutable graph results::
```

File: src/sage/graphs/graph.py [hunk 9]
```diff
@@ -3746,9 +3752,10 @@ def chromatic_number(self, algorithm="DLX", solver=None, verbose=0,
             sage: graphs.RandomBipartite(50,50,0.7).chromatic_number()                  # needs numpy
             2
 
-        A complete multipartite graph with k parts has chromatic number `k`::
+        A complete multipartite graph with `k` parts has chromatic number `k`::
 
-            sage: all(graphs.CompleteMultipartiteGraph([5]*i).chromatic_number() == i for i in range(2,5))
+            sage: all(graphs.CompleteMultipartiteGraph([5]*i).chromatic_number() == i
+            ....:     for i in range(2,5))
```
Comment (gmou3) [inline src/sage/graphs/graph.py]:
```suggestion
            ....:     for i in range(2, 5))
```

File: src/sage/graphs/graph.py [hunk 10]
```diff
@@ -5031,7 +5038,7 @@ def minor(self, H, solver=None, verbose=0, induced=False, *, integrality_toleran
         This function will raise an exception in this case::
 
             sage: g = graphs.RandomGNP(20,.5)
```
Comment (gmou3) [inline src/sage/graphs/graph.py]:
```suggestion
            sage: g = graphs.RandomGNP(20, .5)
```

File: src/sage/graphs/graph.py [hunk 11]
```diff
@@ -5354,7 +5361,7 @@ def eccentricity(self, v=None, by_weight=False, algorithm=None,
             [2, 3, 4]
             sage: G.eccentricity([7,8,9], with_labels=True) == {8: 3, 9: 4, 7: 2}
```
Comment (gmou3) [inline src/sage/graphs/graph.py]:
```suggestion
            sage: G.eccentricity([7, 8, 9], with_labels=True) == {8: 3, 9: 4, 7: 2}
```

File: src/sage/graphs/graph.py [hunk 12]
```diff
@@ -5366,59 +5373,59 @@ def eccentricity(self, v=None, by_weight=False, algorithm=None,
             sage: G = Graph([(0,1,1), (1,2,1), (0,2,3)])
```
Comment (gmou3) [inline src/sage/graphs/graph.py]:
```suggestion
            sage: G = Graph([(0, 1, 1), (1, 2, 1), (0, 2, 3)])
```

File: src/sage/graphs/graph.py [hunk 13]
```diff
@@ -5366,59 +5373,59 @@ def eccentricity(self, v=None, by_weight=False, algorithm=None,
...
             sage: G.eccentricity(algorithm = 'BFS')
```
Comment (gmou3) [inline src/sage/graphs/graph.py]:
```suggestion
            sage: G.eccentricity(algorithm='BFS')
```

File: src/sage/graphs/generators/distance_regular.pyx [hunk 14]
```diff
@@ -2703,11 +2703,11 @@ def distance_regular_graph(list arr, existence=False, check=True):
...
-      - ``True`` - if a graph with the given intersection array exists;
+      - ``True`` -- if a graph with the given intersection array exists;
 
-      - ``False`` - if there is no graph with the given intersection array;
+      - ``False`` -- if there is no graph with the given intersection array;
 
-      - ``Unknown`` - if Sage doesn't know if such a graph exists.
+      - ``Unknown`` -- if Sage doesn't know if such a graph exists.
 
     - ``check`` -- boolean (optional); if ``True``, then checks that the result
```
Comment (gmou3) [inline src/sage/graphs/generators/distance_regular.pyx]:
```suggestion
    - ``check`` -- boolean (default: ``True``); if ``True``, then checks that the result
```

File: src/sage/graphs/generators/distance_regular.pyx [hunk 15]
```diff
@@ -2703,11 +2703,11 @@ def distance_regular_graph(list arr, existence=False, check=True):
...
       of this function has the given intersection array. Default: ``True``
```
Comment (gmou3) [inline src/sage/graphs/generators/distance_regular.pyx]:
```suggestion
      of this function has the given intersection array
```

**Review by gmou3 — CHANGES_REQUESTED** [review]

**Comment by mkoeppe** [comment]
Thanks for the edits!

**Comment by gmou3** [comment]
No problem! (`graphs` is quite a mess I have the impression)

**Review by gmou3 — APPROVED** [review]

**Pull request merged**